deploy/git/website_hash.txt

```diff
@@ -0,0 +1 @@
+b943707
```

Comment:
please revert too

Reply:
This also happened because I had accidentally manually added them. Reverted.